src/components/ModalContent/SwitchNetwork/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import MetamaskIcon from 'assets/img/icons/icon_metamask_color.svg'
+import cn from 'classnames/bind'
+import style from './SwitchNetwork.mod.scss'
+
+const cx = cn.bind(style)
+
+const logoStyle = {
+  width: 100,
+  height: 100,
+}
+
+const SwitchNetwork = ({ closeModal, targetNetwork }) => (
+  <div className={cx('switchNetwork')}>
+    <button className={cx('closeButton')} onClick={() => closeModal()} />
+    <img src={MetamaskIcon} alt="logo" style={logoStyle} />
+    <h3 className={cx('switchNetworkHeading')}>Switch to the {targetNetwork} Network</h3>
```

Comment:
a classname of `heading` would make this more readable, and inside the scss file you can simply nest a `.heading` rule inside the `.switchNetwork` style and it won't interfere with global css.